src/04-modality-specific-files/05-task-events.md

```diff
@@ -129,16 +197,17 @@ of example.
    "identifier": {
        "LongName": "Unique identifier from Karolinska (KDEF) database",
        "Description": "ID from KDEF database used to identify the displayed image"
-   }
+   },
+    "StimulusPresentation": {
+        "OperatingSystem": "Linux-Ubuntu 18.04.5 LTS",
+        "SoftwareName": "Psychtoolbox",
+        "SoftwareRRID": "SCR_002881",
+        "SoftwareVersion": "3.0.14",
+        "Code": "10.5281/zenodo.3581316"
```

Comment:
Let's make this an actual URI so nobody has to parse more than the first four characters to determine it:

```suggestion
        "Code": "doi:10.5281/zenodo.3581316"
```